### DISCUSSION TITLE
plot_memory_usage() not working for me

### DISCUSSION THREAD

**Discussion by brucelaughlin** [OpenDrift/opendrift#1390] [General]
Error is posted immediately below; calling script posted below that:

11:27:38 INFO    opendrift:78: Returning <class 'opendrift.models.oceandrift.OceanDrift'> object
Traceback (most recent call last):
  File "/home/blaughli/tracking_project/utility_scripts/plot_memory", line 32, in <module>
    o.plot_memory_usage()
  File "/home/blaughli/opendrift/opendrift/models/basemodel/__init__.py", line 4481, in plot_memory_usage
    plt.plot(self.memory_usage)
             ^^^^^^^^^^^^^^^^^
AttributeError: 'OceanDrift' object has no attribute 'memory_usage'. Did you mean: 'plot_memory_usage'?

Script:

import sys
import opendrift

tracking_output_file = sys.argv[1]
o = opendrift.open(tracking_output_file)
o.plot_memory_usage()

Called with the line:
plot_memory tracking_output_calcDT_060_saveDT_1440_buffer_100_nSeed_020_startNudge_000000.nc

**Comment by knutfrode** [comment]
Hi, the memory usage is not saved to the netCDF-file, so you can only call this method/plot directly after making the simulation.
Perhaps in the future it might be included in the netCDF-file.

**Reply by brucelaughlin** [reply]
Hi Knut, I figured that this was the case.  I added o.plot_memory_usage() to my scripts, so that the plots will be made while the simulations are still in memory.  Thanks.
